delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/resources/FetchResource.scala

```diff
@@ -0,0 +1,33 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.resources
+
+import ch.epfl.bluebrain.nexus.delta.rdf.IriOrBNode.Iri
+import ch.epfl.bluebrain.nexus.delta.sdk.model.Fetch.FetchF
+import ch.epfl.bluebrain.nexus.delta.sdk.resources.model.{Resource, ResourceState}
+import ch.epfl.bluebrain.nexus.delta.sourcing.Transactors
+import ch.epfl.bluebrain.nexus.delta.sourcing.model.{ProjectRef, ResourceRef}
+import ch.epfl.bluebrain.nexus.delta.sourcing.state.ScopedStateGet
+import doobie.implicits._
+import doobie.{Get, Put}
+
+trait FetchResource {
```

Comment:
You could declare it in the distage module and use directly where it is used instead of passing along the transactors which is a bit low level ?
Any blocker to that ?